py-scripts/lf_report.py

```diff
@@ -792,17 +783,17 @@ def build_table(self):
         self.html += self.dataframe_html
 
     def pass_failed_build_table(self):
-        self.dataframe_html = self.dataframe.style.hide_index(subset=None, level=None, names=False).applymap \
-            (self.pass_fail_background).to_html(index=False,
-                                                justify='center')  # have the index be able to be passed in.
+        self.dataframe_html = self.dataframe.style.hide_index(subset=None, level=None, names=False).applymap(
+                              self.pass_fail_background).to_html(index=False,
+                                                                 justify='center')  # have the index be able to be passed in.
         self.html += self.dataframe_html
 
     def save_csv(self, file_name, save_to_csv_data):
         save_to_csv_data.to_csv(str(self.path_date_time) + "/" + file_name)
 
     def save_pie_chart(self, pie_chart_data):
         explode = (0, 0.1)
-        pie_chart = pie_chart_data.plot.pie(y='Pass/Fail', autopct="%.2f%%", explode=explode, figsize=(10, 10),
+        pie_chart = pie_chart_data.plot.pie(y='Pass/Fail', autopct="%.2f%%", explode=explode, figsize=(10, 10),             # noqa E841 - unused variable
```

Comment:
Does this pass w/ all the whitespace after the conditional? If so, let's just get it to two spaces after, as flake checks for minimum of